package.json

```diff
@@ -13,7 +13,7 @@
     "test-no-git-diff": "git diff --exit-code || exit 1",
     "doc": "lerna run doc",
     "lint": "eslint packages/**/src --no-error-on-unmatched-pattern --ext .ts,.tsx -c .eslintrc.json",
-    "build": "BABEL_ENV=production && lerna run build",
```

Comment:
Do you really need to remove that flag? This change could impact all coins.
Can you maybe apply it in your local `package.json`?

Reply:
that said, i think the initial code is not working at all. 🤔

Reply:
Hi! This change was commited by @valpinkman because I had some issues with tests after migration to TypeScript.

Reply:
:handshake: